.travis.yml

```diff
@@ -155,6 +155,15 @@ jobs:
                             --disable-test-cpp --enable-test-vald --enable-test-helper --enable-test-perf --enable-test-perf-proc --enable-test-example
                           - make -j $(nproc)
                 - stage: test
+                  compiler: "\"clang-3.8 --target=aarch64-linux-gnu\""
+                  env: TEST="clang-3.8 aarch64-linux-gnu" CROSS_ARCH="arm64"
```

Comment:
Do we have a list of available clang compiler levels? In particular clang 4.0 introduced a number of changes that required fix patches in the x86 code paths. I'm wondering if this level would cause cross-compilation issues as well.